src/components/Button/button.scss

```diff
@@ -5,6 +5,11 @@ $button: #fff;
   background-color: $btn;
   border-radius: 4px;
   color: $button;
+}
+.b-button {
+  background-color: #f3994a;
```

Comment:
Why aren't using the `$btn: #f3994a;` variable here?

Reply:
I have implemented that.